File: IntegerListFromUsp/src/Exercicio07.java
import java.util.Scanner;

public class Exercicio07 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        scanner.reset();
        System.out.print("Digite um número: ");
        int numero = scanner.nextInt();
        int acumulador = 0;

        for (int i = 0; i <= numero; i++) {
            if (i % 2 == 0) {
                acumulador += i;
            }
        }

        System.out.println("A soma dos números pares é: %d".formatted(acumulador));
    }
}

File: CPP/IntegerListFromUsp/src/Exercicio07.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cout << "Quantidade de números: ";
    cin >> n;

    long long soma = 0;
    for (int i = 0; i < n; i++) {
        int numero;
        cout << "Número " << (i + 1) << ": ";
        cin >> numero;
        if (numero % 2 == 0) {
            soma += numero;
        }
    }

    cout << "Soma dos pares: " << soma << endl;
    return 0;
}
